elements/drag-drop-elements/info.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
To match the convention over other pl elements, I think the name of the folder should be `pl-drag-drop` rather than `drag-drop-elements`.

Reply:
Also, I'm not sure if it was still up in the air about calling it `pl-drag-drop` or `pl-parsons`, but I prefer to stick with the `pl-drag-drop` name because it can be used for more than just parsons problems.

Reply:
I think the team meeting a few weeks ago said they wanted to carry over the parsons name

personally I'm OK with either, the drag drop name was originally conceived as a placeholder, because I thought it was neat that `drag drop block` rhymed 😄